src/main/java/explorableviz/authoringassistant/paragraph/ExpressionCategory.java

```diff
@@ -7,6 +7,7 @@ public enum ExpressionCategory {
     DATA_RETRIEVAL("Data Retrieval"),
     DIFFERENCE("Difference"),
     TOTAL("Total");
+    MINMAX("Min Max")
```

Comment:
Let's call this "Min/Max" rather than "Min Max" -- given that there is already something (unrelated) in ML called `min-max`.